.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "engines/KataGo-raw"]
 	path = engines/KataGo-raw
 	url = git@github.com:lightvector/KataGo.git
+[submodule "engines/KataGo-tensorflow"]
```

Comment:
To check I understand: this is an older version of KataGo-custom before training switched to PyTorch?

There's no way of doing TensorFlow training with later versions of KataGo-custom?

Having three versions to keep track of is going to be tricky but if it's unavoidable we can manage it.

Reply:
yeah, the tensorflow training/shuffling code has all been removed in KataGo v1.12. We need to either have this extra `engines/KataGo-tensorflow` submodule, or copy all the old tensorflow code back into `KataGo-custom`